Header header
sensor_msgs/LaserScan scan
float32[] xs
float32[] ys
